m/configDTOutputNetCDFL2.m

```diff
@@ -620,7 +648,131 @@
     'grid_coordinates' []
     'grid_method'      [] };
 
-
+  %% Variables specific to SeaExplorer
+  var_attr_list.mfl_v1 = {
+      'long_name'     'MiniFluo raw path 1'
+      'standard_name' 'tryptophan_or_pyrene_measurement'
+      'units'         'counts'
+      'coordinates'   'time depth latitude longitude'
+      '_FillValue'    default_fill_value
+      'sources'       []
+      'conversion'       []
+      'grid_sources'     []
+      'grid_coordinates' []
+      'grid_method'      [] };
+  
+  var_attr_list.mfl_v2 = {
+      'long_name'     'MiniFluo raw path 2'
+      'standard_name' 'phenantrene_or_fluoren_measurement'
```

Comment:
Although they are synonyms, it seems that the standard name
is pyren**e** with the final *e* ([[1]], [[2]] and [[3]]).
Is the same for fluoren**e** and phenantren**e**.

  [1]: <https://en.wikipedia.org/wiki/Pyrene>
  [2]: <http://www.commonchemistry.org/ChemicalDetail.aspx?ref=129-00-0>
  [3]: <https://www.ebi.ac.uk/chebi/searchId.do?chebiId=39106>